src/components/mgt-person/mgt-person.ts

```diff
@@ -480,22 +480,22 @@ export class MgtPerson extends MgtTemplatedComponent {
       return;
     }
```

Comment:
`mgt-person` has a check for personDetails, but no image. Make sure to do something similar in `mgt-person-card`.

Reply:
Done